README.md

```diff
@@ -62,3 +73,32 @@ To view the metrics and traces port-forward the services:
     kubectl port-forward jaeger-0 16686
 
 Then navigate to http://localhost:9090 for metrics and http://localhost:16686 for traces.
+
+## Simulation
+
+To run a simulation delete the old jobs and re-apply the simulation job definitions:
+
+    kubectl delete jobs.batch simulate-manager simulate-worker-{0..9}
+    kubectl apply -k k8s/ceramic/
+
+> NOTE: We will need the k8s operator to make job configuration dynamic.
+> This is a manual method in the meantime.
+
+## Analysis
+
+To analyze the results of a simulation first copy the metrics-TIMESTAMP.parquet file from the otel-0 pod.
+First restart otel-0 pod so it writes out the parquet file footer.
+
+    kubectl delete pod otel-0
+    kubectl cp otel-0:data/metrics-TIMESTAMP.parquet ./analyze/metrics.parquet
+    cd analyze
```

Comment:
In your demo you had to do an additional step to find the `TIMESTAMP` should that step be described here?